tests/Giraffe.Tests/Giraffe.Tests.fsproj

```diff
@@ -41,13 +41,20 @@
     <PackageReference Include="Microsoft.AspNetCore.TestHost" Version="7.0.*" />
  </ItemGroup>
 
+ <ItemGroup Condition=" '$(TargetFramework)' == 'net8.0'">
+    <PackageReference Include="Microsoft.AspNetCore.TestHost" Version="8.0.*" />
+ </ItemGroup>
```

Comment:
Why are you adding this package for .NET 8?

Reply:
it is added for each framework version just above. Otherwise, the package used for testing will not be referenced in .net 8 at all.